README.md

```diff
@@ -43,13 +43,14 @@ via CI.
 ```shell
 docker run --rm -v "$(pwd)"/bin/:/builder/bin -it openwrt/sdk
 # inside the Docker container
+[ ! -d ./scripts ] && ./setup.sh
```

Comment:
Whats the idea behind this? Can you explain it in the Git commit description?

Reply:
If you download a snapshot container it only contains the setup

Reply:
I know about that, wondering mostly about that `[ ! -d ./scripts ]` part.

Reply:
This is a one-shot use, ephemeral container - you exit, and it's gone. Every time you enter you need to run `setup.sh`, unless the user has customised the container somehow. In any case, it's an improvement, since the `setup.sh` needs to be run, which wasn't documented yet.

Reply:
An advantage is that you can copy paste those instructions into a shell prompt multiple times when an image is up without the penalty of running setup.

Reply:
Can you explain it in the Git commit description?